scss/_patterns_navigation.scss

```diff
@@ -100,31 +127,37 @@
       }
 
       .p-navigation__link {
```

Comment:
Yes please do create an issue. We need to external link to use the currentColor.